canvas_modules/common-canvas/src/command-actions/createAutoNodeAction.js

```diff
@@ -27,12 +30,10 @@ export default class CreateAutoNodeAction extends Action {
 		if (this.apiPipeline.isLinkNeededWithAutoNode(this.newNode, this.srcNode)) {
 			this.newLink = this.apiPipeline.createLink(this.newNode, this.srcNode);
 		}
-		this.newPipeline = null;
-		if (this.newNode && this.newNode.open_with_tool === "shaper") {
```

Comment:
@tomlyn Do you remember what this was for?

Reply:
It was when we supported the shaper in modeler flows.  It should never really have been in common-canvas because common canvas should not know about other tools that edit the pipeline flow.